app/commands/git/sync_authors_and_contributors.rb

```diff
@@ -3,17 +3,39 @@ class SyncAuthorsAndContributors
     include Mandate
 
     def call
+      sync_exercise_authors_and_contributors!
+      sync_concept_authors_and_contributors!
+    end
+
+    private
+    def sync_exercise_authors_and_contributors!
       ::Exercise.find_each do |exercise|
         begin
-          SyncAuthors.(exercise)
+          SyncExerciseAuthors.(exercise)
+        rescue StandardError => e
+          Rails.logger.error "Error syncing exercise authors for #{exercise.track.slug}/#{exercise.slug}: #{e}"
```

Comment:
Should these be opened as track issues? If not in this PR, should we add a TODO here (and to the other places) for them?)

Reply:
Probably. Let me look into that